Classes/KIFUITestActor.m

```diff
@@ -444,7 +444,7 @@ - (void)expectView:(UIView *)view toContainText:(NSString *)expectedResult
         KIFTestWaitCondition([actual isEqualToString:expected], error, @"Failed to get text \"%@\" in field; instead, it was \"%@\"", expected, actual);
         
         return KIFTestStepResultSuccess;
-    } timeout:1.0];
+    } timeout:2.0];
```

Comment:
FYI in our code we bumped the timeout for text views/fields to 10s. Timeouts should be long (even 30s) to avoid flaky tests like it was the case before. On certain (slower) build machines and longer texts entered, even 2s might not be enough, and you'll be in the same place as before.